brownie-config.yaml

```diff
@@ -21,3 +21,5 @@ reports:
     - contracts/test/TestGuestList.sol
   exclude_contracts:
     - SafeMath
+    - SafeERC20
+    - Address
```

Comment:
Removed SafeERC20 and Address from coverage report as they are not a Yearn contracts (they're from OpenZeppelin)